format-specs/geoparquet.md

```diff
@@ -1,5 +1,47 @@
+# Geospatial Parquet format
 
-## A Geospatial Parquet format
+## Overview
 
-This page is a scratch pad for ideas to define a 'geo-parquet' format, aligning on a default schema and format for geospatial data in Parquet.
+The [Apache Parquet][parquet] provides a standardized open-source columnar storage format. This specification defines how geospatial data
+should be stored in parquet format, including the representation of geometries and the required additional metadata.
 
+## Geometry columns
+
+Geometry columns are stored using the `BYTE_ARRAY` parquet type. They are encoded as WKB.
```

Comment:
Is there an official specification for WKB to link to?

Reply:
Another question related to WKB is also which "flavour" of WKB is used (eg EWKB). I personally don't really know how important this is though (eg for GeoPandas, being backed by GEOS for WKB parsing, that automatically parses different flavors).